.cursor/rules.mdc

```diff
@@ -0,0 +1,54 @@
+---
+description: General rules for cursor
+alwaysApply: true
+---
+
+# Cursor Rules for Konflux Architecture Repository
+
+## Commit Message Rules
+
+See [commit-rules.mdc](commit-rules.mdc).
+
+## General Guidelines
+
+1. **Incremental Changes**: Make small, testable changes
+2. **User Feedback**: Incorporate user feedback and iterate on solutions
+3. **Error Handling**: Provide clear error messages and debugging information
+4. **Documentation**: Keep architecture documentation up-to-date 
+
+## Language Style Guidelines
+
+1. **Be direct**: Use clear, specific, and factual language. Avoid exaggerated, sensational, or vague descriptions.
+2. **Be plain**: Avoid "hype words" in language style and "nothingburger" statements.
+
+## Code Quality Rules
+
+Run tests locally with `make`.
```

Comment:
Do we even have tests for this repo?

Reply:
We do! https://github.com/konflux-ci/architecture/blob/main/Makefile